envs/__init__.py

```diff
@@ -1,2 +1,22 @@
-from .env_base import EnvironmentBase
+from .env_base import Environment
 from .env_4inrow import Env4InRow
+
+__all__ = ['Environment', 'create_env']
```

Comment:
The `create_env` function should allow the user to easily create an environment using just its name,  
like `'4-in-row`. The user shouldn't remember the name of the class `Env4InRow` and be able to access it's implementation, it's pointless.  
We put the `Environment` base class at `__all__` because we want the user to be able to determine what functions on the environment are there to serve his needs.